### ISSUE TITLE
Captions (subtitle) justification other than centered?

### ISSUE THREAD

**Issue by wjmt3** [mackworth/cTiVo#347]
The closed captions, as included in the original show, _aren't_ centered, but they are in the final output. Is there any way to change this? It's extremely annoying. At first, I thought it was just a VLC option, but changing settings there doesn't make a difference. And QuickTime Player doesn't have any options to set, but still shows the subtitles as centered. Based on that, I figure it is something cTivo is doing? Thanks!

**Comment by mackworth** [comment]
SRT files are how ccextractor specifies captions to MP4, and do not have the ability to specify positions for each liine . There is a flag for the overall positioning (mdia.minf.stbl.stsd.tx3g.horizontalJustification), and cTiVo does currently specify center. 
Unfortunately, I tried changing it to left and it had no effect on either QuickTime Player or VLC. 

It sounds like you found what I did, that VLC does have such controls, but they don't seem to work on the Mac at all.
Not sure what else I can do. Maybe complaining to VLC?
